cg/meta/workflow/balsamic.py

```diff
@@ -648,6 +648,16 @@ def get_variant_callers(self, case_id: str) -> list[str]:
                 )
         return analysis_var_callers
 
+    def get_pons(self, case_id: str) -> list[str]:
+        """Return list of panel of normals used for analysis."""
+        pons: list[str] = []
+        analysis_metadata: BalsamicAnalysis = self.get_latest_metadata(case_id)
+        if analysis_metadata.config.panel:
+            pon_cnn: str | None = analysis_metadata.config.panel.pon_cnn
+            if pon_cnn:
```

Comment:
you loose the type hint though, but I think it is still clear if you want to implement it
```suggestion
            if pon_cnn := analysis_metadata.config.panel.pon_cnn
```